deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/dido/kustomization.yaml

```
@@ -31,4 +31,4 @@ patchesStrategicMerge:
 images:
   - name: storetheindex
     newName: 407967248065.dkr.ecr.us-east-2.amazonaws.com/storetheindex/storetheindex
-    newTag: 20221029095953-db83b7c9fab3615621063378fdda568c6e8ba209
+    newTag: 0.5.0 # {"$imagepolicy": "storetheindex:storetheindex:tag"}
```

Do not include the comment. That has a special meaning to the CD controllers.
```suggestion
    newTag: 0.5.0
```